### DISCUSSION TITLE
Version of ArchR hg38 genome?

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#1658] [Questions / Documentation]
Hi @rcorces,

Would you be able to share what specific version of h38 precompiled ArchR uses? 

I see that this is made from BSgenome.Hsapiens.UCSC.hg38, TxDb.Hsapiens.UCSC.hg38.knownGene, org.Hs.eg.db according to documentation. 

I ask because there is a lncRNA missing from this precompiled annotation compared to the current UCSC genome browser.

ArchR gene track (using hg38 precompiled default) at chr2:118797677−118857678:

<img width="783" alt="Screen Shot 2022-09-30 at 5 36 04 PM" src="https://user-images.githubusercontent.com/54336249/193359551-b7a62001-bde0-4b4b-9660-b63268129d60.png">

UCSC track at chr2:118797677−118857678:
<img width="1671" alt="Screen Shot 2022-09-30 at 5 38 15 PM" src="https://user-images.githubusercontent.com/54336249/193359832-9fec51c1-9b74-421f-a52c-26a20f6ca1ac.png">

I think the discrepancy here between ArchR and UCSC may be caused by a difference in the annotation set and/or version (UCSC perhaps showing a newer version which happens to add in this lncRNA LINC01956). Thus, distal peaks in this region (left of EN1) would presumably become intronic if this lncRNA LINC01956 was included in the annotation.

Thanks for your help!

**Comment by rcorces** [comment]
Your UCSC genome browser is showing Gencode genes. Those are quite a bit different from the genes described by the TxDB object `TxDb.Hsapiens.UCSC.hg38.knownGene`
